include/fastdds/rtps/transport/UDPTransportDescriptor.h

```diff
@@ -17,43 +17,53 @@
 
 #include <fastdds/rtps/transport/SocketTransportDescriptor.h>
 
-namespace eprosima{
-namespace fastdds{
-namespace rtps{
+namespace eprosima {
+namespace fastdds {
+namespace rtps {
 
 /**
  * UDP Transport configuration
  *
- * - bufferSize:    length of the buffers used for transmission. Passing
- *                  a buffer of different size will cause transmission to
- *                  fail.
+ * - m_output_udp_socket: port number for the outgoing messages.
+ *
+ * - non_blocking_send: do not block on send operations. When it is set to true, send operations will return
+ * immediately if the buffer is full, but no error will be returned to the upper layer. This means that the
+ * application will behave as if the datagram is sent and lost.
  *
- * - interfaceWhiteList: Lists the allowed interfaces.
  * @ingroup TRANSPORT_MODULE
  */
-typedef struct UDPTransportDescriptor: public SocketTransportDescriptor
+struct UDPTransportDescriptor : public SocketTransportDescriptor
 {
-   virtual ~UDPTransportDescriptor(){}
-
-   RTPS_DllAPI UDPTransportDescriptor();
-
-   RTPS_DllAPI UDPTransportDescriptor(const UDPTransportDescriptor& t);
-
-   uint16_t m_output_udp_socket;
-
-   /**
-    * Whether to use non-blocking calls to send_to().
-    *
-    * When set to true, calls to send_to() will return inmediately if the buffer is full, but
-    * no error will be returned to the upper layer. This means that the application will behave
-    * as if the datagram is sent but lost (i.e. throughput may be reduced). This value is
-    * specially useful on high-frequency best-effort writers.
-    *
-    * When set to false, calls to send_to() will block until the network buffer has space for the
-    * datagram. This may hinder performance on high-frequency writers.
-    */
-   bool non_blocking_send = false;
-} UDPTransportDescriptor;
+    //! Destructor
+    virtual ~UDPTransportDescriptor() = default;
+
+    //! Constructor
+    RTPS_DllAPI UDPTransportDescriptor();
+
+    //! Copy constructor
+    RTPS_DllAPI UDPTransportDescriptor(
+            const UDPTransportDescriptor& t);
```

Comment:
Could we use `= default` here?

Reply:
Done in e81f0cfc11d2b5f33cd1a96728bd45e4068d28b7